runtime/service.go

```diff
@@ -565,7 +565,12 @@ func (s *service) mountDrives(requestCtx context.Context) error {
 // If the VM has not been created yet and the timeout is hit waiting for it to exist, an error will be returned
 // but the shim will continue to shutdown.
 func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMRequest) (_ *empty.Empty, err error) {
-	defer logPanicAndDie(s.logger)
+	defer func() {
+		// we ignore the error here due to cleanup will only succeed if the jailing
+		// process was killed via SIGKILL
+		s.jailer.Close()
+		logPanicAndDie(s.logger)
+	}()
```

Comment:
Signals can be coming from anywhere. If calling Close() is necessary, how about putting the invocation in monitorVMExit()?

Reply:
Done. Moved to `monitorVMExit`